00_SETUP.md

```diff
@@ -51,9 +36,31 @@ being able to run anything related to conda.
 
 On Windows, open the `Anaconda Prompt` terminal app.
 
-Now navigate to this directory in the terminal. For example, if you installed
+## 3. Install git (if needed)
+
+At the prompt opened in the previous step, enter this command to see whether git is already installed and accessible to this shell:
+
+    % git --version
+
+If the output shows a git version, proceed to the next step.  Otherwise install git by entering the following command and following the prompts:
+
+    % conda install git
+
+## 4. Clone This Repository
+
+Download the workshop folder using
+[git](https://help.github.com/articles/set-up-git/):
+
+    % git clone https://github.com/NASA-NAVO/aas_workshop_2020_winter.git
+
+## 3. Create a conda environment for the workshop
```

Comment:
Need to renumber this section and the following ones to 5, 6, 7.

Reply:
Thanks for the review! It figures I would mess up the counting.

Fortunately the Astropy workshop  environment didn't install directly from git, so shouldn't strictly need the git install as much as this one.